Dockerfile

```diff
@@ -39,6 +40,23 @@ RUN set -x && \
         dnf -y install /rpms/*.rpm && \
         dnf -y autoremove; \
     fi && \
+    # generate certificates that will be used for the testing purposes
+    /usr/local/bin/x509certgen x509KeyGen ca && \
+    /usr/local/bin/x509certgen x509KeyGen server && \
+    /usr/local/bin/x509certgen x509KeyGen client && \
+    /usr/local/bin/x509certgen x509KeyGen ca2 && \
+    /usr/local/bin/x509certgen x509KeyGen server2 && \
+    /usr/local/bin/x509certgen x509KeyGen client2 && \
+    /usr/local/bin/x509certgen x509SelfSign ca && \
+    /usr/local/bin/x509certgen x509SelfSign ca2 && \
+    /usr/local/bin/x509certgen x509CertSign --CA ca server && \
+    /usr/local/bin/x509certgen x509CertSign --CA ca client && \
+    /usr/local/bin/x509certgen x509CertSign --CA ca2 server2 && \
+    /usr/local/bin/x509certgen x509CertSign --CA ca2 client2 && \
+    # configure httpd
+    sed -i "s/#ServerName .*/ServerName ${HOSTNAME}:80/" /etc/httpd/conf/httpd.conf && \
```

Comment:
HOSTNAME variable is exported by bash itself... This change helps to get rid off the message 'httpd: Could not reliably determine the server's fully qualified domain name'